fava/static/javascript/charts.js

```diff
@@ -348,7 +346,7 @@ class BarChart extends BaseChart {
       .tickSizeOuter(0);
 
     this.yAxis = axisLeft(this.y)
-      .tickFormat(format('.2s'));
+      .tickFormat(num => formatCurrency(num, '.2s'));
```

Comment:
The `'.2s'` is needed for the chart-y-labels, that's why the `fstr` is a parameter.